.eslintrc.js

```diff
@@ -65,8 +65,19 @@ module.exports = {
 		{
 			files: ['*.ts'],
 			parser: '@typescript-eslint/parser',
-			plugins: ['@typescript-eslint'],
-			extends: ['plugin:@typescript-eslint/recommended', 'prettier/@typescript-eslint', 'plugin:prettier/recommended'],
+			extends: [
+				'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
+				'plugin:import/typescript',
+				'plugin:prettier/recommended', // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+			],
+			parserOptions: {
+				ecmaVersion: 2020, // Allows for the parsing of modern ECMAScript features
+				sourceType: 'module', // Allows for the use of imports
+			},
+			rules: {
+				'import/extensions': 'never',
```

Comment:
I think this would be `'import/extensions': ['error', 'never']` if we want to always show an error on missing extensions. Or `'import/extensions': 'off'` to disable the check completely. Depends on what we want to achieve.